app/quality/checkstyle/checkstyle.xml

```diff
@@ -34,7 +34,7 @@
         <module name="SuppressWarningsHolder" />
 
         <!-- Checks for Javadoc comments. See http://checkstyle.sf.net/config_javadoc.html -->
-        <module name="JavadocType">
+        <!--<module name="JavadocType">
```

Comment:
👀

Reply:
hehe I deleted this so that I don't have to add javadocs comments to every single class before commiting